.idea/codeStyles/Project.xml

```diff
@@ -5,6 +5,22 @@
       <option name="NAMES_COUNT_TO_USE_IMPORT_ON_DEMAND" value="999" />
     </JavaCodeStyleSettings>
     <JetCodeStyleSettings>
+      <option name="PACKAGES_TO_USE_STAR_IMPORTS">
```

Comment:
we don't need any change in this file. Remove this file from the PR. 
Let me know if you need help with git commands in this.

Reply:
I have no idea why it keeps adding itself. Everytime I save the project in AS it gets added. Please help

Reply:
@Victor-Titan please don't delete this file.